src/frontend/components/SøknadsSteg/EøsSteg/Søker/useEøsForSøker.tsx

```diff
@@ -39,6 +38,12 @@ export const useEøsForSøker = (): {
     idNummerFelter: Felt<string>[];
 } => {
     const { søknad, settSøknad, tekster } = useApp();
+    const {
+        arbeidNorge,
+        hvorBor,
+        pensjonNorge: pensjonNorge1,
+        utbetalinger,
+    } = tekster().EØS_FOR_SØKER;
```

Comment:
Vi har også en enum for denne, ESanitySteg

Reply:
Kompilatoren skjønner at det er enumet jeg bruker. Tror jeg synes dette er finere en `tekster()[ESanitySteg.EØS_FOR_SØKER]`, men åpen for begge deler 🤷‍♂️
<img  width="785" alt="image" src="https://user-images.githubusercontent.com/17828446/196941520-0c050efa-0130-40fc-942e-dc9c2a5589c9.png">

Reply:
Oki! Tenkte kanskje den ikke skjønte det, men da er det jo good :D

Reply:
Enig i at det er penere som du hadde skrevet det da i såfall!